### PR TITLE
✅  FitText: Add Preact unit tests

### PR DESCRIPTION
**Pull request by caroqliu**
Related to #28281

### CODE REVIEW

File: extensions/amp-fit-text/1.0/component.js [hunk 1]
```diff
@@ -78,8 +78,16 @@ export function FitText({
...
+      <div
+        ref={measurerRef}
+        className={classes.fitTextContent}
+        part="container"
+      >
+        <div
+          ref={heightRef}
+          className={classes.minContentHeight}
+          part="content"
+        >
```
Comment (krdwan) [inline extensions/amp-fit-text/1.0/component.js:90]: Should we use the part attribute here as a way of selecting these elements for testing?

I have only seen them being used for CSS purposes so far and wondering if we should do this from a code cleanliness pespective?

Comment (caroqliu) [reply extensions/amp-fit-text/1.0/component.js:90]: Good point, I revised the test to look for the classes instead and removed these additions. Thanks for the recommendation!

Comment (krdwan) [reply extensions/amp-fit-text/1.0/component.js]: Cool, thanks for making the change!

**Review by krdwan — APPROVED** [review]
LGTM

**Pull request merged**